src/components/Header/Link/Link.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 
-function HeaderLink(props) {
+function HeaderLink(props: any) {
```

Comment:
Should we add the correct types in a following PR?

Reply:
A following pr sounds good